### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by samhita-alla**
Signed-off-by: Samhita Alla aallasamhita@gmail.com

[README.md File Preview](https://github.com/samhita-alla/flyte/blob/flyte-readme/README.md)

* Updated Flyte Logo as it isn't visible when the dark mode is enabled. Picked it up from the rsts folder.
* Included HTML in the markdown for components which need to be center-aligned. 
* Re-phrased the `Introduction` section.
* Included links to documentation in the `Features` and `Available Plugins` section.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,68 +1,109 @@
...
+Here's a quick visual tour of the console.
     
 ![Flyte console Example](https://github.com/flyteorg/flyte/raw/static-resources/img/first-run-console-2.gif)
 
-Refer to [Docs - Getting Started](https://docs.flyte.org/en/latest/index.html) for complete end to end example.
+To dig deeper into Flyte, refer to the [Documentation](https://docs.flyte.org/en/latest/index.html).
+
+<html>
+<h2 id="community--resources"> 
+  ❄️ Community & Resources
```
Comment (cosmicBboy) [inline README.md:91]: maybe a more welcoming emoji here? 🤝

File: README.md [hunk 2]
```diff
@@ -72,86 +113,87 @@ Resources that would help you get a better understanding of Flyte.
...
- - Single Task Execution support - Start executing a task and then convert it to a workflow
- - gRPC / REST interface to define and executes tasks and workflows
- - Type safe construction of pipelines, each task has an interface which is characterized by its input and outputs. Thus illegal construction of pipelines fails during declaration rather than at
-   runtime
- - Types that help in creating machine learning and data processing pipelines like - Blobs (images, arbitrary files), Directories, Schema (columnar structured data), collections, maps etc
+- MLOps Coffee session - [Flyte: an open-source tool for scalable, extensible, and portable workflows](https://anchor.fm/mlops/episodes/MLOps-Coffee-Sessions-12-Flyte-an-open-source-tool-for-scalable--extensible---and-portable-workflows-eksa5k)
+
+<html>
+<h2 id="features"> 
+  🔥 Features
```
Comment (cosmicBboy) [inline README.md:127]: What do you think this ordering of the main sections?

- Introduction
- Current Deployments
- Quick Start
- Features
- Component Repos
- Community & Resources
- Top Contributors

The rationale is to immediately show users where it's being used, how they can start, and what the key features are, then progressively go deeper into details and how they can contribute

**Comment by kumare3** [comment]
@samhita-alla here are some other logos that are available - https://drive.google.com/drive/folders/1KU-CPO5HzO1VrroFJAAS-TLxqTLpZR6T
Also, We have started moving all static resources to flyte's untracked branch. @EngHabu what do you think of just creating a new repo with static resources?

File: README.md (hunk 2 above)
Comment (kumare3) [reply README.md:127]: I think we should also add a status of the platform - like production ready etc.
Reason: Some folks have asked me - when will you be 1.0.0? We should probably explain that we are production ready.
Another thing we should probably think is if we should just release a 1.0.0 at some point soon

Comment (SandraGH5) [reply README.md:127]: @cosmicBboy Would the Current Deployments section include Available Plugins, Current Usage and Production K8s Operators? 
Or would these be in other sections?

File: README.md (hunk 1 above)
Comment (samhita-alla) [reply README.md:91]: Done.

File: README.md (hunk 2 above)
Comment (samhita-alla) [reply README.md:127]: @SandraGH5 -- I think it'd be better if we include those in the separate sections.

@cosmicBboy  -- How about moving swapping `Current Deployments` and `Quick Start` sections? People usually would want to estimate how easy it is to get Flyte up and running.

Here's the final TOC:

- Introduction
- Quick Start
- Current Deployments
- Features
- Available Plugins
- Component Repos
- Production K8s Operators
- Community & Resources
- Top Contributors

Let me know if this is okay to proceed with.

@kumare3 -- We're already including that in the tagline below the Flyte logo -- "Flyte is a **production-grade**, container-native, type-safe workflow and pipelines platform optimized for large scale processing and machine learning written in Golang". Will this suffice?

**Comment by samhita-alla** [comment]
> @samhita-alla here are some other logos that are available - https://drive.google.com/drive/folders/1KU-CPO5HzO1VrroFJAAS-TLxqTLpZR6T
> Also, We have started moving all static resources to flyte's untracked branch. @EngHabu what do you think of just creating a new repo with static resources?

How about we modify the logo (if required) once the folder containing the static resources is available?

**Comment by kumare3** [comment]
@samhita-alla reconcile with #853 ?

**Review by kumare3 — APPROVED** [review]
🙏🏽

**Pull request merged**